### DISCUSSION TITLE
How to set app.update.langpack.enabled

### DISCUSSION THREAD

**Discussion by LokoOn** [mozilla/policy-templates#877] [Q&A] [answered]
Hello all,

according to this article https://firefox-source-docs.mozilla.org/toolkit/mozapps/update/docs/BackgroundUpdates.html app.update.langpack.enabled need to set to false to get the background updates process working.
We are using the admx files to enable app.update.background.enable and app.update.auto settings, but I could not find any settings for app.update.langpack.enabled 
Is there a way, to deploy such not yet implemented config through a "free text" config via GPO or is the only way to deploy a policies.json file to the client?

Greetings,
Loko

**Comment by mkaply** [comment, marked as answer]
Yes, all of the app.update preferences can be set via the Preferences GPO:

https://github.com/mozilla/policy-templates/blob/master/README.md#preferences

**Reply by LokoOn** [reply]
Hi Michael,
thank you for your assistance!
